<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debugInfo</class>
 <widget class="QWidget" name="debugInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>344</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="infoLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="1" column="1">
     <widget class="QTableView" name="tableView">
       <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
         <bool>true</bool>
       </property>
     </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="connections_button">
     <property name="text">
      <string>Connections:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="connections">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QRadioButton" name="objects_button">
     <property name="text">
      <string>Objects:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="objects">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Known Peers:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="peers">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="about">
       <property name="text">
        <string>About HyperBit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aboutQt">
       <property name="text">
        <string>About Qt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="configureNetwork">
       <property name="text">
        <string>Configure Network</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
 <connection>
   <sender>connections_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>debugInfo</receiver>
   <slot>show_connections(bool)</slot>
  </connection>
 </connections>
</ui>
